### PR TITLE
Some style and expression change of README file

### PR DESCRIPTION
**Pull request by FHU-yezi**
I found some spaces in black lines, remove this.

Some headers doesn't comes with a black line at the bottom, fix this.

I just installed the wrong package, so add a notice in README.

The `bun` install will cost 1-3 minutes, and we have a small problem when init project for the first time: the output say that we are prepare demo project files, but actually we are downloading and installing `bun`, add a notice about this.

`Node.js` is the official name, not `NodeJS`.

### CODE REVIEW

**Comment by Alek99** [comment]
> I found some spaces in black lines, remove this.
> 
> Some headers doesn't comes with a black line at the bottom, fix this.
> 
> I just installed the wrong package, so add a notice in README.
> 
> The `bun` install will cost 1-3 minutes, and we have a small problem when init project for the first time: the output say that we are prepare demo project files, but actually we are downloading and installing `bun`, add a notice about this.
> 
> `Node.js` is the official name, not `NodeJS`.

just curious what computer os are you running on because on mine its much faster than 1-3 minutes

**Review by Alek99 — COMMENTED** [review]
just curious what computer/os are you running on because on mine its much faster than 1-3 minutes, more like 30 sec for me

**Comment by FHU-yezi** [comment]
> just curious what computer/os are you running on because on mine its much faster than 1-3 minutes

I didn't time it, but it will spend more time that add some files to a path will cost.

My computer is a bit old and it run Manjaro Linux on a HDD.

**Comment by Alek99** [comment]
> > just curious what computer/os are you running on because on mine its much faster than 1-3 minutes
> 
> I didn't time it, but it will spend more time that add some files to a path will cost.
> 
> I saw a blinking process bar with a progress increasing, it is the progress bar of bun installation?
> 
> My computer is a bit old and it run Manjaro Linux on a HDD.

Ah ok lets take the time out of this for now and I'll do some investigation and get an accurate estimate

File: README.md [hunk 1]
```diff
@@ -28,15 +27,17 @@ Pynecone is a full-stack Python framework that makes it easy to build and deploy
...
 * Python 3.7+
-* [NodeJS 12.22.0+](https://nodejs.org/en/)
+* [Node.js 12.22.0+](https://nodejs.org/en/) (Don't worry, you'll never have to write any Javascript)
 
 ```
 $ pip install pynecone-io
 ```
 
-## Create your first Pynecone app
+(Not `pynecone`, it's `pynecone-io`, IO for Input/Output)
```
Comment (picklelo) [inline README.md:36]: This is a good reminder, can we remove the last part of it? Rephrase it to:
```
*NOTE:* Make sure you install the correct package: `pynecone-io`, not `pynecone`.
```

File: README.md [hunk 2]
```diff
@@ -46,13 +47,15 @@ $ cd my_app_name
 $ pc init
 ```
 
-This initializes a template app in your new directory.
+When you run this command for the first time, we will download and install `bun` automatically, it will spend 1-3 minutes to finish.
```
Comment (picklelo) [inline README.md:50]: +1 on removing the 1-3 minutes part (it should on average take less).

**Review by picklelo — CHANGES_REQUESTED** [review]

**Review by picklelo — APPROVED** [review]
Looks good!
